### DISCUSSION TITLE
How to enable recordings on FA54 Main Unit

### DISCUSSION THREAD

**Discussion by rheinlein** [AxisCommunications/discussions#191] [VAPIX]
Hello, 

I have written a python program to configure an axis FA54 main unit with 3 sensors from the ground up. (We are deploying 800 of them)

The only thing I cannot figure out is how to enable recording on each of the sensors via the API. I have read the documentation and reached out to support (who directed me here). Any suggestions would be greatly appreciated as I don't want to have to manually turn on recordings for 800 units.

**Comment by onie5** [comment]
There is a video capture API which you can use to write pull streams and write to file. The sample capture code is in C++ not sure how you work it out in Python. You can also try out OpenCV.

**Comment by vivekatoffice** [comment]
Related link:
[203](https://github.com/orgs/AxisCommunications/discussions/203)
